docs/reference/early-access.md

```diff
@@ -0,0 +1,56 @@
+---
+id: early-access
+title: Early access
+sidebar_label: Early access
+description: "Use early access releases to quickly learn about upcoming features and components."
+---
+
+Use early access releases to quickly learn about upcoming features and components.
+
+## Types of early access releases
+
+Selected Camunda features and components are released upfront as alpha or beta versions, e.g., to provide users the opportunity to test a feature and participate in the development by giving feedback prior to market launch (general availability, GA).
+
+Early access releases are made available outside of the [release policy](/reference/release-policy.md).
+
+### Alpha
+
+Alpha features and components are introduced to gather early feedback from users. These features and components are still under heavy development, and might change. These releases are intended only for non-production usages in trying out recent, potentially yet unfinished new features. Characteristics are:
+
+- Not for production use
+- Unstable, potentially causing performance issues
+- APIs, dependencies and configuration are likely to change
+- Not necessarily feature-complete
+- May not be documented
+- No update support guaranteed
+- Not covered by service level agreements (SLAs), and no support obligations
+
+### Beta
+
+The scope of beta features and components is close to that of the targeted GA release. A beta release may still be missing some features. Characteristics are:
+
+- Not for production use
+- Unstable, potentially causing performance issues
+- APIs, dependencies and configuration are not likely to change
+- Potentially feature-complete
+- Update support provided
+- Not covered by service level agreements (SLAs), and no support obligations
+
+:::note
+
+Even though there are no support obligations, customers can still provide feedback through their designated support channels, file issues in the respective [GitHub repositories](https://github.com/camunda), as well as submit questions and suggestions in the [Camunda Forum](https://forum.camunda.io/).
+
+:::
+
+### General Availability (GA)
+
+All features and components launched on the market that are not considered early access are considered generally available. Characteristics are:
```

Comment:
I don't think a developer/user would consider themselves the "market".
How does the user know whether a feature is launched "on the market"?

Reply:
Good comment, I'll try to rephrase